llpc/context/llpcCompiler.cpp

```diff
@@ -846,7 +846,7 @@ Result Compiler::buildPipelineWithRelocatableElf(Context *context, ArrayRef<cons
       auto data = reinterpret_cast<const char *>(elfBin.pCode);
       elf[stage].assign(data, data + elfBin.codeSize);
       LLPC_OUTS("Cache hit for shader stage " << getShaderStageName(static_cast<ShaderStage>(stage)) << "\n");
-      stageCacheAccesses[stage] = CacheAccessInfo::CacheHit;
+      stageCacheAccesses[stage] = CacheAccessInfo::InternalCacheHit;
```

Comment:
We should double check.  I just realized that if `userShaderCache` is not nullptr, then it could be an application cache that is being checked.

Reply:
I checked the XGL https://github.com/GPUOpen-Drivers/xgl.
As far as I checked, `pipelineInfo->pShaderCache` is never set by XGL and it should be `nullptr`.

Reply:
Updated